tempesta_fw/str.h

```diff
@@ -206,15 +202,16 @@ size_t tfw_ultoa(unsigned long ai, char *buf, unsigned int len);
  * @eolen	- the length of string's line endings, if present (as for now,
  *		  it should be 0 if the string has no EOL at all, 1 for LF and
  *		  2 for CRLF);
- * @flags	- 3 most significant bytes for number of chunks of compound
- * 		  string and the least significant byte for flags;
+ * @chunkn  	- 3 most significant bytes for number of chunks of compound string;
+ * @flags	- flags;
  */
 typedef struct {
 	void		*ptr;
 	struct sk_buff	*skb;
 	unsigned long	len;
 	unsigned char	eolen;
-	unsigned int	flags;
+	unsigned int	chunkn : __TFW_STR_CN_BITS;
```

Comment:
I'd like to propose to rename field to make it differ from `chunks` more. I believe that it's easier to confuse `chunks` with `chunkn` than `nchunks` with `chunks`, for example.